public/app/features/dashboard-scene/scene/types.ts

```diff
@@ -11,18 +12,12 @@ export interface DashboardLayoutManager extends SceneObject {
    * @param isEditing
    */
   editModeChanged(isEditing: boolean): void;
-  /**
-   * We should be able to figure out how to add the explore panel in a way that leaves the
-   * initialSaveModel clean from it so we can leverage the default discard changes logic.
-   * Then we can get rid of this.
-   */
-  cleanUpStateFromExplore?(): void;
```

Comment:
I'm trying to understand, but I don't get it. Probably because I don't understand the problem described. How do we solve this now?

Reply:
@ivanortegaalba  - I think it was just a leftover from https://github.com/grafana/grafana/pull/94458

Reply:
yea, left over